TwListView.js

```diff
@@ -11,7 +11,14 @@ const styles = StyleSheet.create({
 // disabled for now, at least until we figure out if this
 // will have state or not
 /* eslint-disable react/prefer-stateless-function */
+/**
```

Comment:
Hm. I didn't think it would be necessary to add the docstrings in this file if you didn't otherwise change it. Travis would have let it pass at least

Reply:
The git on my computer seems to convert the line endings to \r\n which lint doesn't allow. Therefore I changed the line endings -> then I had to add documentation. 
I'm looking into fixing the line ending issue.